widgetssdk/src/main/java/com/glia/widgets/core/secureconversations/domain/SecureConversationTopBannerVisibilityUseCase.kt

```diff
@@ -0,0 +1,50 @@
+package com.glia.widgets.core.secureconversations.domain
+
+import com.glia.androidsdk.Engagement.MediaType
+import com.glia.androidsdk.queuing.Queue
+import com.glia.androidsdk.queuing.QueueState
+import com.glia.widgets.core.queue.QueueRepository
+import com.glia.widgets.core.queue.QueuesState
+import io.reactivex.rxjava3.core.Flowable
+
+/**
+ * Should show top banner in secure conversations chat only if both conditions are met
+ * - There is an open queue with text, audio or video media
+ * - Secure conversation engagement is started
```

Comment:
I think something like `has pending secure conversations' will more accurately reflect the use case.

Reply:
Fixed in [here](https://github.com/salemove/android-sdk-widgets/compare/7c13ae4315595dd52bc9379b6984ac4df1b736bf..c6afdc8be35f28ce248a6db3a50790e54e1c9a6e)